src/components/ProductCard.jsx

```diff
@@ -1,7 +1,30 @@
+import { useCart } from "../context/cart-context";
+
 export const ProductCard = ({ product }) => {
-  const { _id, title, src, price, mrp, rating } = product;
+  const {
+    cartState: { cartItems },
```

Comment:
you can destrcuture this in the context itself so that you don't have to destructure in every component while using useCart

Reply:
In some places, I needed the cartState. So, I've kept it that way.